CatchMe/CatchMe/Source/ViewControllers/CharacterVC.swift

```diff
@@ -109,12 +169,10 @@ extension CharacterVC: UITableViewDelegate {
                 self.upperView.snp.updateConstraints { make in
                     make.height.equalTo(width-offset)
```

Comment:
- 하면 띄워주는게 좋을 거 같아요!
```suggestion
                    make.height.equalTo(width - offset)
```